readme.md

```diff
@@ -343,6 +343,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 - [Linux Containers](https://github.com/Friz-zy/awesome-linux-containers)
 - [Beginner-Friendly Projects](https://github.com/MunGell/awesome-for-beginners)
 - [Bluetooth Beacons](https://github.com/beaconinside/awesome-beacon)
+- [Spanish NLP](https://github.com/dav009/awesome-spanish-nlp)
```

Comment:
I don't think NLP is a common enough acronym to use here, can you use the full form instead?